src/js/engagement_view/src/modules/uploads/plugins/apiRequests.tsx

```diff
@@ -30,7 +30,7 @@ export const deletePlugin = async ( pluginName: string ): Promise <boolean> => {
             body: JSON.stringify({plugins_to_delete: [pluginName]})
         }
     );
-    const body = await res.json();
+    await res.json();
     return true;
```

Comment:
Seems like we should be checking the response for success, not just returning true.